cookiecutter-django-ida/{{cookiecutter.repo_name}}/Makefile

```diff
@@ -96,17 +93,25 @@ migrate: ## apply database migrations
 html_coverage: ## generate and view HTML coverage report
 	coverage html && open htmlcov/index.html
 
+# Define PIP_COMPILE_OPTS=-v to get more information during make upgrade.
+PIP_COMPILE = pip-compile --rebuild --upgrade $(PIP_COMPILE_OPTS)
```

Comment:
Is `--rebuild` necessary here? Clearing caches seems like it would make things take longer than necessary.

Reply:
Removed the `--rebuild` flag as suggested since we don't have any explicit need for it.